content/actions/learn-github-actions/using-workflow-templates.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: Using workflow templates
-shortTitle: Using templates
-intro: '{% data variables.product.product_name %} provides workflow templates for a variety of languages and tooling.'
+title: Using starter workflows
```

Comment:
@pulkitaggarwl you will need  to rename file as well to using-starter-workflows.md